payroll-processor-client/src/app/employee/employee-list/state/employee-list.service.ts

```diff
@@ -37,4 +37,20 @@ export class EmployeeListService {
         complete: () => this.store.setLoading(false),
       });
   }
+
+  updateEmployeeStatus(id: string, employee: EmployeeUpdate) {
+    this.store.setLoading(true);
+    return this.http
```

Comment:
You will generally want to either A) return the `Observable<T>` from the HTTP request or B) `.subscribe()`.

In this case you are returning the `Subscription` (disposable resource) to the caller.

Returning the `Observable<T>` gives the caller more control over how the stream is handled, and the ability to participate in the event stream (react to a new event).

Calling `.subscribe()` in the service means the caller of this method (`updateEmployeeStatus`) will need to be informed about the results of the HTTP request some other way (Akita does this by exposing a Query class which is all events emitted by the store).